index.js

```diff
@@ -32,9 +39,21 @@ getFullNames(usersArray);
 
 const getUsersCreditDetails = arr => {
   // Your code goes here ...
+  let usersCreditDetails = [];
+  for (let user of arr) {
+    let obj = {};
+  const {firstName, lastName, balance} = user;
+  const userDetails = {
+    firstName,
+    lastName,
+    balance
+  }
+  usersCreditDetails.push(userDetails);
```

Comment:
you could have also pushed directly the object 
```js
userCreditDetails.push({firstName, lastName, balance})
```

But the way you wrote is also correct